go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -328,12 +352,35 @@ func (be *BuiltinBackupEngine) ExecuteBackup(ctx context.Context, cnf *Mycnf, my
 		if err := WaitForSlaveStart(mysqld, slaveStartDeadline); err != nil {
 			return usable, vterrors.Wrap(err, "slave is not restarting")
 		}
-	}
 
-	// And set read-only mode
-	logger.Infof("resetting mysqld read-only to %v", readOnly)
-	if err := mysqld.SetReadOnly(readOnly); err != nil {
-		return usable, err
+		// wait for reliable seconds behind master
```

Comment:
doc nit: "Wait for a reliable 'seconds behind master' value" or something. My initial read of this was that it was going to wait for a number of seconds that we considered reliable to have us get a new seconds behind master reading.

Reply:
I'll fix the comment